File: src/classWithMethodsAndFields/JDGUI/PersonDecompiled/Person.java
package classWithMethodsAndFields.JDGUI.PersonDecompiled;


import kotlin.Metadata;
import org.jetbrains.annotations.NotNull;

@Metadata(mv = {2, 0, 0}, k = 1, xi = 82, d1 = {"\000\036\n\002\030\002\n\002\020\000\n\000\n\002\020\016\n\000\n\002\020\b\n\002\b\003\n\002\020\002\n\000\030\0002\0020\001B\027\022\006\020\002\032\0020\003\022\006\020\004\032\0020\005\006\004\b\006\020\007J\006\020\b\032\0020\tR\016\020\002\032\0020\003X\004\006\002\n\000R\016\020\004\032\0020\005X\004\006\002\n\000\006\n"}, d2 = {"LclassWithMethodsAndFields/Person/Person;", "", "name", "", "age", "", "<init>", "(Ljava/lang/String;I)V", "sayHello", "", "KotlinDecompiler"})
public final class Person {
    @NotNull
    private final String name;

    private final int age;

    public Person(@NotNull String name, int age) {
        this.name = name;
        this.age = age;
    }

    public final void sayHello() {
        System.out.println("Привет, меня зовут " + this.name);
    }
}

File: src/classWithMethodsAndFields/Fernflower/PersonCodeConvert/Person.kt
package classWithMethodsAndFields.Fernflower.PersonCodeConvert

import org.jetbrains.annotations.NotNull // Not imported by CodeConvert

/*
@Metadata(
    mv = [2, 0, 0],
    k = 1,
    xi = 48,
    d1 = ["\u0000\u001e\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0000\n\u0002\u0010\u000e\n\u0000\n\u0002\u0010\b\n\u0002\b\u0002\n\u0002\u0010\u0002\n\u0000\u0018\u00002\u00020\u0001B\u0015\u0012\u0006\u0010\u0002\u001a\u00020\u0003\u0012\u0006\u0010\u0004\u001a\u00020\u0005¢\u0006\u0002\u0010\u0006J\u0006\u0010\u0007\u001a\u00020\bR\u000e\u0010\u0004\u001a\u00020\u0005X\u0082\u0004¢\u0006\u0002\n\u0000R\u000e\u0010\u0002\u001a\u00020\u0003X\u0082\u0004¢\u0006\u0002\n\u0000¨\u0006\t"],
    d2 = ["LclassWithMethodsAndFields/Person;", "", "name", "", "age", "", "(Ljava/lang/String;I)V", "sayHello", "", "KotlinDecompiler"]
)

 */
class Person(
    @NotNull val name: String, // NotNull is not imported
    val age: Int
) {
    fun sayHello() {
        val message = "Привет, меня зовут $name"
        println(message)
    }
}

